third_party/libjpeg-turbo/BUILD.gn

```diff
@@ -306,6 +306,13 @@ static_library("libjpeg") {
     defines -= [ "NO_PUTENV" ]
   }
 
+  if (is_clang_16) {
```

Comment:
https://source.corp.google.com/search?q=is_clang_16&sq=&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2F23.lts.1%2B

There's no definition for is_clang_16 in 23.lts

We should cherrypick the change that introduces it first

Reply:
This one perhaps?

https://lbshell-internal-review.git.corp.google.com/c/cobalt_src/+/276300

Reply:
This pr is merging into trunk, I'l cherry pick the relevant changes into 23.lts later.